sdk/appconfiguration/Azure.Data.AppConfiguration/samples/README.md

```diff
@@ -11,74 +11,10 @@ description: Samples for the Azure.Data.AppConfiguration client library
 
```

Comment:
Is the 
``page_type: sample
languages:
- csharp``

header needed here, like the one in KeyVault, or will that be added automatically?

Reply:
Sorry, I don't understand. In KeyVault Secrets (https://github.com/Azure/azure-sdk-for-net/edit/master/sdk/keyvault/Azure.Security.KeyVault.Secrets/samples/README.md): 
```
---
page_type: sample
languages:
- csharp
products:
- azure
- azure-key-vault
name: Azure.Security.KeyVault.Secrets samples for .NET
description: Samples for the Azure.Security.KeyVault.Secrets client library
---
```

In this file:
```
---
page_type: sample
languages:
- csharp
products:
- azure
- azure-app-configuration
name: Azure.Data.AppConfiguration samples for .NET
description: Samples for the Azure.Data.AppConfiguration client library
---
```

Reply:
Nevermind - my mistake.